books/RayTracingInOneWeekend.html

```diff
@@ -1391,19 +1391,20 @@
 We need some math constants that we conveniently define in their own header file. For now we only
 need infinity, but we will also throw our own definition of pi in there, which we will need later.
 There is no standard portable definition of pi, so we just define our own constant for it. We'll
-throw common useful constants and future utility functions in `rtweekend.h`, our general main header
-file.
+also throw common useful constants and future utility functions in `rtweekend.h`, our general main
+header file.
```

Comment:
Nit: This is a bit wordy

```
We'll also throw useful common constants, and future utility functions in `rtweekend.h`. 
This will be our general main header file.
```